pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/ConnectionTest.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.pinot.client;
```

Comment:
add license header.
You can run `mvn license:format -T1C`

Reply:
Thanks!